site/docs/voice/bxml/startTranscription.mdx

```diff
@@ -0,0 +1,236 @@
+---
+id: startTranscription
+title: Start Transcription
+slug: /voice/bxml/startTranscription
+description: A general overview of Bandwidth's startTranscription BXML Verb
+keywords:
+  - bandwidth
+  - voice
+  - bxml
+  - start
+  - transcribing
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The StartTranscription verb allows a segment of a call to be transcribed and for the developing transcription to be sent off to another destination for additional processing.
+
+When used on a call, live transcription updates for one or both sides (tracks) of the call will be sent to the specified destination. The transcription will continue until the call ends or the [`<StopTranscription>`][1] verb is used. A total of 4 concurrent track transcriptions are allowed on a call. A `<StartTranscription>` request that uses `both` tracks will count as 2 concurrent transcriptions.
+A call has only two tracks, which are named after the direction of the media from the perspective of the Programmable Voice platform:
+- `inbound`: media received by Programmable Voice from the call executing the BXML;
+- `outbound`: media sent by Programmable Voice to the call executing the BXML.
+
+Note that this has no correlation to the direction of the call itself. For example, if either an inbound or outbound call is being transcribed and executes a `<SpeakSentence>`, the `inbound` track will be the callee's audio and the `outbound` track will be the text-to-speech audio.
```

Comment:
What text-to-speech audio? I think this needs to be cleaned up

Reply:
The TTS audio generated by `SpeakSentence`

Reply:
I completely glossed that this example was talking about SpeakSentence. Ignore me.